app/scripts/controllers/overview.js

```diff
@@ -1101,6 +1101,7 @@ function OverviewController($scope,
     // - API objects by binding name
     state.bindingsByApplicationUID = {};
     state.applicationsByBinding = {};
+    state.validBindingsByApplicationUID = {};
```

Comment:
im not in love with the variable names here or in service-row, open to other suggestions